google/api/annotations.proto

```diff
@@ -104,7 +111,7 @@ extend google.protobuf.MethodOptions {
 // Useful to be able to declare a project as a referent for another field.
 // (See `resources.proto` in this directory.)
 message Project {
-  string name = 1 [(google.api.resource).path = "projects/*"];
+  string name = 1 [(google.api.resource).path = "projects/{project_id}"];
```

Comment:
This should be a file level resource definition in order to avoid creating a message that is unneeded.

```proto
option (google.api.resource_definition) = {
  name: "Project",
  path: "projects/{project_id}"
}
```

Reply:
After this is done, would we be able to move these resource definitions to `google/api/common_resources.proto` or is there still a potential import issue if we change this to resource definitions.